components/compositing/webview_renderer.rs

```diff
@@ -290,14 +290,19 @@ impl WebViewRenderer {
                 let cached_hit_test_result = match event.event {
```

Comment:
Please call this `hit_test_result` as it isn't always the cached hit test any longer.
```suggestion
                let hit_test_result = match event.event {
```